lib/data/nextcloud/file_syncer.dart

```diff
@@ -161,17 +161,45 @@ abstract class FileSyncer {
   static Future uploadFileFromQueue() async {
     if (_isUploadingFile) return;
     await _uploadQueue.waitUntilLoaded();
-    if (_uploadQueue.value.isEmpty) return;
-
+    if (_uploadQueue.value.isEmpty) {
+      return;
+    }
+    if (_isUploadingFile){
```

Comment:
These `FileSyncer` changes look like duplicates of #1081. If nothing new is here (?), I'll just disregard it when I rebase this branch.
(Also please avoid updating the branch via merge commits as it makes the commit history harder to read)

Reply:
Yes, I forgot to create new branch. I see that you already closed #1081.